owners/OWNERS.example.yaml

```diff
@@ -58,17 +59,40 @@
 - "*.{js,css}": frontend
 - "package.{,lock.}json": somebodyelse
 
+# Prepending a user or team name with a `?` will prevent this owner from being
```

Comment:
Line 58: Would it be useful to provide a reference link to the brace expansion syntax used by the parser?

Reply:
Done!